components/governance/org.wso2.carbon.governance.generic.ui/src/main/resources/web/generic/edit_ajaxprocessor.jsp

```diff
@@ -40,192 +41,194 @@
-    }
-
-    String content = client.getArtifactContent(relativePath);
-    if(content ==null){
-        return;
-    }
-    OMElement data = GenericUtil.loadAddedServiceContent(content);
-    GenericUIGenerator gen = new GenericUIGenerator(dataName, dataNamespace);
-    OMElement uiconfig = gen.getUIConfiguration(client.getArtifactUIConfiguration(
-            request.getParameter("key")),request,config,session);
-    request.setAttribute("content",data);
-    Iterator widgets = uiconfig.getChildrenWithName(new QName(null, UIGeneratorConstants.WIDGET_ELEMENT));
-    StringBuffer table = new StringBuffer();
-    while(widgets.hasNext()){
-        OMElement widget = (OMElement)widgets.next();
-        String widgetText = gen.printWidgetWithValues(widget, data, false, true, true, request, config).replace("\n", "<!--LF-->").replace("\r", "<!--CR-->");
-        table.append(widgetText.replace("<!--LF-->", "\n").replace("<!--CR-->", "\r"));
-    }
-
-    List<Map> validatationAttributes = gen.getValidationAttributes(uiconfig);
-    String[] mandatory = gen.getMandatoryIdList(uiconfig);
-    String[] name = gen.getMandatoryNameList(uiconfig);
-    String[] unboundedNameList = gen.getUnboundedNameList(uiconfig);
-    String[] unboundedWidgetList = gen.getUnboundedWidgetList(uiconfig);
-    String[] unboundedTooltipList = gen.getUnboundedTooltipList(uiconfig);
-    String[][] unboundedValues = gen.getUnboundedValues(uiconfig, request, config);
-    String[][] dateIdAndNameList = gen.getDateIdAndNameList(uiconfig, data, true);
-%>
-
-<br/>
-<script type="text/javascript">
-
-    jQuery(document).ready(
-            function() {
-                initCollapseMenu("#_addEditTable");
-            }
-    );
-</script>
-
-<script type="text/javascript">
-    function addEditArtifact(){
-        sessionAwareFunction(function() {
-            getArtifactName();
-            var reason = "";
-        <%for(int i=0;i<mandatory.length;i++){%>
-            reason += validateEmpty(document.getElementById('<%=mandatory[i]%>'),
+        OMElement data = GenericUtil.loadAddedServiceContent(content);
+        GenericUIGenerator gen = new GenericUIGenerator(dataName, dataNamespace);
+        OMElement uiconfig = gen
+                .getUIConfiguration(client.getArtifactUIConfiguration(request.getParameter("key")), request, config,
+                        session);
+        request.setAttribute("content", data);
+        Iterator widgets = uiconfig.getChildrenWithName(new QName(null, UIGeneratorConstants.WIDGET_ELEMENT));
+        StringBuffer table = new StringBuffer();
+        while (widgets.hasNext()) {
+            OMElement widget = (OMElement) widgets.next();
+            String widgetText = gen.printWidgetWithValues(widget, data, false, true, true, request, config)
+                    .replace("\n", "<!--LF-->").replace("\r", "<!--CR-->");
+            table.append(widgetText.replace("<!--LF-->", "\n").replace("<!--CR-->", "\r"));
```

Comment:
They have been there since moved to github and I did the reformatting of those lines.